src/pyatmo/thermostat.py

```diff
@@ -146,26 +148,30 @@ def __init__(self, auth: NetatmoOAuth2, home_id: str):
             self.rooms[room["id"]] = room
 
         for module in self.raw_data.get("modules", []):
-            if module["type"] == "NATherm1":
-                thermostat_id = module["id"]
-                if thermostat_id not in self.thermostats:
-                    self.thermostats[thermostat_id] = {}
+            self._store_module(module)
+
+    def _store_module(self, module: Dict) -> None:
```

Comment:
you have two method with the same name (`_store_module`), this could be confusing

Reply:
I'll close this and try to come up with more clever refactoring after yours are merged.